esp8266-weather-station-color.ino

```diff
@@ -238,15 +260,38 @@ uint8_t currentTouchPoint = 0;
 void loop() {
   static bool asleep = false; //  asleep used to stop screen change after touch for wake-up
   gfx.fillBuffer(MINI_BLACK);
-  if (touchController.isTouched(0)) {
+
+  /* Break up the screen into 4 sections a touch in section:
+   * - Top changes the time format
+   * - Left back one page
+   * - Right forward one page
+   * - Bottom jump to page 0
+   */
+  const static uint8_t top = gfx.getHeight() / 4;
```

Comment:
Kind-of unrelated to this PR but...

Is it worth the effort fetching width & height dynamically? `TouchControllerWS.cpp` has 240/320 embedded in the code 🙈 Maybe extract that into `settings.h`, WDYT?

Reply:
I don't think that hard coding the screen size is good. The screen is defiantly something that should be runtime calibrated if it is a fixed size why would touch calibration be needed?